samples/nrf9160/secure_services/README.rst

```diff
@@ -31,7 +31,7 @@ Building and running
 .. include:: /includes/build_and_run.txt
```

Comment:
This should be `.. include:: /includes/build_and_run_nrf9160.txt` as in the other nRF9160 samples. Then we don't need line 33 and 34, because it's part of the included text.

Reply:
Modified.